api/src/main/resources/workbench-api.yaml

```diff
@@ -1697,6 +1697,30 @@ paths:
           description: HTML Read Only version of the notebook
           schema:
             "$ref": "#/definitions/ReadOnlyNotebookResponse"
+  "/v1/admin/workspaces/{workspaceNamespace}/{workspaceId}/notebooks/{notebookName}/readonly":
+    parameters:
+      - "$ref": "#/parameters/workspaceNamespace"
+      - "$ref": "#/parameters/workspaceId"
+    post:
+      tags:
+        - workspaceAdmin
+      description: Get the HTML for a read only view of the notebook. RESEARCHER_DATA_VIEW authority required.
+      operationId: adminReadOnlyNotebook
+      parameters:
+        - in: body
+          name: accessReason
+          required: true
+          schema:
+            "$ref": "#/definitions/AccessReason"
+        - in: path
+          name: notebookName
+          required: true
+          type: string
+      responses:
+        200:
+          description: HTML Read Only version of the notebook
+          schema:
+            "$ref": "#/definitions/ReadOnlyNotebookResponse"
```

Comment:
Is the html base64 encoded or something? How do you get it to play nice with JSON otherwise?